backend/src/test/java/de/neuefische/backend/service/DoctorServiceTest.java

```diff
@@ -80,4 +85,230 @@ void addExistingDoctorTest() {
         assertEquals("Doctor Linda Holder, Dentist, Bonn, already exists in the database", thrown.getMessage());
         verify(doctorRepo).existsDoctorByFirstNameAndLastNameAndSpecialtyAndCity("Linda", "Holder", "Dentist", "Bonn");
     }
+
+
+    @Test
+    @DisplayName("addAppointment should add a new appointment to the database")
+    void addAppointmentTest() {
+        // GIVEN
+        DoctorDto expectedDoctorDto = DoctorDto.builder()
```

Comment:
I really like moving the construction of the Objects needed for the tests in their own methods, preferably in a testutil class, that can be accessed from multiple testclasses where eventually  the same objects will be needed for testing. Makes your code look much less cluttered and saves you a lot of typing/copyPasting